terraform-cli/terraform-cli-task.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: terraform-cli
+spec:
+  inputs:
+    resources:
+      - name: source
+        type: git
+    params:
+      - name: ARGS
+        description: The terraform cli commands to tun
+        type: array
+        default:
+          - "--help"
+      - name: terraform-secret
+        description: The terraform secret with credentials
+        type: string
+        default: "terraform-creds"
+  steps:
+    - name: init
+      image: quay.io/rcmendes/terraform-cli:latest
```

Comment:
Wed rather use official image, if it need to use a custom user image, it would be best if we have the Dockerfile in the repository (i.e: like s2i or openshift-cli task). Thanks!

Reply:
I think there's a official image from hashicorp, but I created one using an UBI image, how can I submit this image to the official repository? meanwhile I'll test the hashicorp official image for this

Reply:
You just need to add the Dockerfile in this repo i.e: https://github.com/tektoncd/catalog/blob/master/s2i/Dockerfile

Reply:
Understood, will add that, and will add the Dockerfile for the ansible-tower-cli task as well in another PR.

Reply:
Done, Dockerfile added :+1: